tests/testthat/fixtures/describe.scm

```diff
@@ -0,0 +1,32 @@
+(call
+	function: [
+		(identifier) @parent_function
+		(namespace_operator
+			lhs: (identifier) @pkg (#eq? @pkg "testthat")
```

Comment:
Thanks for the discussion that lead to me learning these predicates could go here, inside the alternation!